core/services/keeper/upkeep_executer.go

```diff
@@ -313,22 +310,37 @@ func buildJobSpec(
 	gasPrice *big.Int,
 	gasTipCap *big.Int,
 	gasFeeCap *big.Int,
-	chainID string,
+	chainId *utils.Big,
 ) map[string]interface{} {
+	evmChainID := ""
+	if chainId != nil {
+		evmChainID = chainId.String()
+	}
+
+	var checkUpkeepGasLimit, performUpkeepGasLimit uint32
+	if !isArbitrum(chainId) {
```

Comment:
nit. can you revert this logic? Arbitrum is the special case to set a different value

Reply:
done